cmd/registry/cmd/util/util.go

```diff
@@ -30,3 +33,17 @@ func TargetConfiguration() (name string, c config.Configuration, err error) {
 
 	return
 }
+
+// FQName ensures the project and location, if available,
+// are properly included in the resource name.
```

Comment:
I think it would be good to spell out "fully qualified" somewhere in the comment.

Reply:
Will do